man/systemctl.xml

```diff
@@ -2477,6 +2469,27 @@ Jan 12 10:46:45 example.com bluetoothd[8900]: gatt-time-server: Input/output err
         </listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--when=</option></term>
+
+        <listitem>
+          <para>When used with <command>halt</command>, <command>poweroff</command>, <command>reboot</command>
+          or <command>kexec</command>, schedule the action to be performed at the given timestamp,
+          which should adhere to the syntax documented in <citerefentry
+          project='man-pages'><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>
+          section "PARSING TIMESTAMPS". Use <option>--cancel</option> to cancel the scheduled action.</para>
+        </listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><option>--cancel</option></term>
+
+        <listitem>
+          <para>When used with <command>halt</command>, <command>poweroff</command>, <command>reboot</command>
+          or <command>kexec</command>, cancel the action scheduled using <option>--when=</option>.</para>
+        </listitem>
```

Comment:
hmm, systemctl has so many options already, can we maybe fold this into `--when=`? i.e. when an empty string is passed, cancel the thing again? While we certainly should support cancelling, I think it's a very nichey feature and hence overloading `--when` like this, sounds totally ok to me.